### DISCUSSION TITLE
We are trending!

### DISCUSSION THREAD

**Discussion by CarterLi** [fastfetch-cli/fastfetch#854] [Show and tell]
![image](https://github.com/fastfetch-cli/fastfetch/assets/6134068/f33af5a3-f45a-450f-a0c1-3eb30bcffc82)

**Comment by ataey** [comment]
Congrats!

**Comment by CrazyWolf13** [comment]
Congrats!!
Yeah, just discovered your project and it looks awesome!!!

**Comment by GoJackzi** [comment]
Awesome!

**Comment by shuvashish76** [comment]
IG It's FOSS has some contribution in it :)
https://news.itsfoss.com/neofetch-rip/ (Fastfetch the 1st alternative they suggested with the Neofetch archival news)
https://itsfoss.com/neofetch-alternatives/ (Fastfetch still their top recommendation among all the alternatives)

See the daily stars graph https://emanuelef.github.io/daily-stars-explorer/#/fastfetch-cli/fastfetch (750stars in just one day i.e. 04-05-2024)

## [Star History](https://star-history.com/)

[![Star History Chart](https://api.star-history.com/svg?repos=fastfetch-cli/fastfetch&type=Date)](https://star-history.com/#fastfetch-cli/fastfetch&Date)

**Reply by shuvashish76** [reply]
Another article : https://itsfoss.com/fine-control-fastfetch/
